compiler/ballerina-parser/src/main/java/io/ballerina/compiler/syntax/tree/NodeParser.java

```diff
@@ -125,4 +125,26 @@ public static TypeDescriptorNode parseTypeDescriptor(String text) {
         BallerinaParser parser = ParserFactory.getParser(text);
         return parser.parseAsTypeDescriptor().createUnlinkedFacade();
     }
+
+    /**
+     * Parses the input as an object member.
+     *
+     * @param text the input
+     * @return a {@code Node}
+     */
+    public static Node parseObjectMember(String text) {
+        BallerinaParser parser = ParserFactory.getParser(text);
+        return parser.parseAsObjectMember().createUnlinkedFacade();
+    }
+
+    /**
+     * Parses the input as a module part.
+     *
+     * @param text the input
+     * @return a {@code ModulePartNode}
+     */
+    public static ModulePartNode parseModulePart(String text) {
+        BallerinaParser parser = ParserFactory.getParser(text);
+        return parser.parseAsModulePart().createUnlinkedFacade();
```

Comment:
`invalidateRestAndAddToTrailingMinutiae()` is not required as the module part is the one we are using to parse a bal file. It will parse the full bal file without leaving any tokens behind. Therefore can use `parse()` directly.
```suggestion
        return parser.parse().createUnlinkedFacade();
```